lib/validator.js

```diff
@@ -47,8 +47,10 @@ class Validator {
     }
   }
 
-  validate (schemaRef, data) {
-    return this.ajv.validate(schemaRef, data)
+  validate (schemaRef, data, errors) {
+    const valid = this.ajv.validate(schemaRef, data)
+    if (this.ajv.errors && Array.isArray(errors)) errors.push(...this.ajv.errors)
```

Comment:
Maybe we should add some reference to `schemaRef` or the errors will be relative to the schema position in the `anyOf` array

Reply:
the error already contains a `schemaPath` property